adam_core/coordinates/cometary.py

```diff
@@ -491,43 +298,77 @@ def from_cartesian(
             covariances=covariances_cometary,
             origin=cartesian.origin,
             frame=cartesian.frame,
-            mu=mu,
         )
 
         return coords
 
+    def to_cometary(self) -> "CometaryCoordinates":
+        return self
+
+    @classmethod
+    def from_cometary(
+        cls, cometary_coordinates: "CometaryCoordinates"
+    ) -> "CometaryCoordinates":
+        return cometary_coordinates
+
```

Comment:
These look deletable. Maybe this is for a type checker though?

Reply:
Deleted.